Dockerfile

```diff
@@ -501,6 +501,96 @@ RUN \
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-libc as sdk-go-fips
+
+ARG ARCH
+ARG TARGET="${ARCH}-bottlerocket-linux-gnu"
+ARG SYSROOT="/${TARGET}/sys-root"
+ARG GOVER="1.21.4"
+ARG AWS_LC_FIPS_VER="2.0.2"
+
+USER root
+RUN DEBIAN_FRONTEND=noninteractive dnf install -y golang cmake xz wget ca-certificates clang python curl git
+
+USER builder
+WORKDIR /home/builder/sdk-go-fips
+COPY ./hashes/go /home/builder/hashes
+RUN \
+  sdk-fetch /home/builder/hashes && \
+  tar --strip-components=1 -xf go${GOVER}.src.tar.gz && \
+      rm go${GOVER}.src.tar.gz
+
+WORKDIR /home/builder/sdk-go-fips
+COPY patches/go-fips/* ./
+RUN git init && git apply --whitespace=nowarn *.patch
+
+# Build aws-lc crypto modules
+USER root
+WORKDIR /boring/boringssl
+COPY ./hashes/aws-lc /home/builder/hashes
+ENV UPSTREAM_SOURCE_FALLBACK=true
+RUN \
+  sdk-fetch /home/builder/hashes && \
+  tar --strip-components=1 -xf AWS-LC-FIPS-${AWS_LC_FIPS_VER}.tar.gz && \
+    rm AWS-LC-FIPS-${AWS_LC_FIPS_VER}.tar.gz
+RUN \
+  install -p -m 0644 -Dt /aws-lc/licenses LICENSE
+
+# Build BoringCrypto.
+WORKDIR /boring
+ARG LANG=C
+ARG ClangV=17
+ARG CFLAGS="-O2 -g -pipe -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions"
+ARG CXXFLAGS=${CFLAGS}
+ARG LDFLAGS="-Wl,-z,relro -Wl,-z,now"
+RUN cp /home/builder/sdk-go-fips/src/crypto/internal/boring/build-boring.sh /boring/build-boring.sh
+RUN sed -i "s|-DGOBORING -fPIC \"\$@\"|--gcc-install-dir=/usr/lib/gcc/'$TARGET'/11/ -DGOBORING -fPIC --target='$TARGET' --sysroot='$SYSROOT' \"\$@\"|g" /boring/build-boring.sh
+RUN /boring/build-boring.sh
```

Comment:
nit: condense these `RUN` statements for consistency with the overall style in this Dockerfile